Assets/Scripts/Player/Input Parser/InputParser.cs

```diff
@@ -11,16 +11,15 @@ public sealed class InputParser : MonoBehaviour
         private const string CANVAS = "Canvas";
         private const string INTERACTABLE_TAG = "Interactable";
         
-        [SerializeField, Range(0, 20)] private float interactableRayDistance;
+        [SerializeField, Range(10, 100)] private float interactableRayDistance;
```

Comment:
```suggestion
        [SerializeField, Range(10, 100)] private float interactableRayDistance = 10;
```
standaard value

Reply:
Je kan toch als je `Range(10, 100)` hebt kan je toch die = 10 assignment weghalen? dan is 10 toch de base value

Reply:
Niet altijd, had bij movement speed ook hetzelfde idee en die zat toen zo van "nee ik ga me speed op 0 zetten" :)

Reply:
@DanielNijkamp heeft wel gelijk tho, default is nu het eerste getal van de `Range()`.

Reply:
Dan waarom deze line in basemovement veranderen?

```
        [SerializeField,Range(1,10)] protected float p_movementSpeed = 1; 
```

Reply:
> Niet altijd, had bij movement speed ook hetzelfde idee en die zat toen zo van "nee ik ga me speed op 0 zetten" :)

Ja maar als je dus geen value onder 10 gaat hebben kan je 10 als base hebben

Reply:
> > > Dan waarom deze line in basemovement veranderen?
> > > ```
> > >         [SerializeField,Range(1,10)] protected float p_movementSpeed = 1; 
> > > ```
> > 
> > 
> > Check reply
> 
> Ja dan deze ook gewoon:
> 
> ```
> [SerializeField,Range(1,10)] protected float p_movementSpeed; 
> ```

Dat was het ook, maar bas heeft het verandered naar: = 1;

Reply:
> > > > Dan waarom deze line in basemovement veranderen?
> > > > ```
> > > >         [SerializeField,Range(1,10)] protected float p_movementSpeed = 1; 
> > > > ```
> > > 
> > > 
> > > Check reply
> > 
> > 
> > Ja dan deze ook gewoon:
> > ```
> > [SerializeField,Range(1,10)] protected float p_movementSpeed; 
> > ```
> 
> Dat was het ook, maar bas heeft het verandered naar: = 1;

@bas-boop habibi wat is dis